Makefile

```diff
@@ -162,11 +162,21 @@ src/asm/parser.c: src/asm/parser.y
 	echo "DEFS=$$DEFS"; \
 	${BISON} $$DEFS -d ${YFLAGS} -o $@ $<
 
+# Only RGBGFX uses libpng (POSIX make doesn't support pattern rules to cover all these)
```

Comment:
Is `+=` syntax in POSIX?

Reply:
Apparently not; `pmake` fails. But, it doesn't even support this:
```make
src/gfx/main.o:          REALCXXFLAGS = `${REALCXXFLAGS} ${PNGCFLAGS}`
[etc]
```

Any ideas? Will we have to spell out every single one?

```make
src/gfx/main.o: src/gfx/main.cpp
	$Q${CXX} ${REALCXXFLAGS} ${PNGCFLAGS} -c -o $@ $<
```

Reply:
Looks like yes. :/

Reply:
Oh well, that's done then.